gui/control_main.py

```diff
@@ -142,7 +142,7 @@ class ControlMain(QtWidgets.QMainWindow):
     roiChangeSignal = QtCore.Signal(int, str)
     highMagCursorChangeSignal = QtCore.Signal(int, str)
     lowMagCursorChangeSignal = QtCore.Signal(int, str)
-    cryostreamTempSignal = QtCore.Signal(str)
+    cryostreamTempSignal = QtCore.Signal(object)
```

Comment:
is there any reason why it couldn't be a float or something?

Reply:
For FMX currently the value returned by the PV is None whereas its a float at AMX. This is a catch all for different values. It is correctly converted to string in the GUI. Once the FMX PV provides correct values we can switch it to float